tests/sync/test_browsertype_connect.py

```diff
@@ -12,13 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import sys
 import time
 
 import pytest
 
 from playwright.sync_api import BrowserType, Error
 from tests.server import Server
 
+pytestmark = pytest.mark.skipif(
```

Comment:
why a warning and what gives us this warning?

Reply:
https://github.com/aaugustin/websockets/issues/1029